README.md

```diff
@@ -153,6 +153,8 @@ loss.backward()
 | Mixtral     | `liger_kernel.transformers.apply_liger_kernel_to_mixtral`  | RoPE, RMSNorm, SwiGLU, CrossEntropyLoss        |
 | Gemma2      | `liger_kernel.transformers.apply_liger_kernel_to_gemma`    | RoPE, RMSNorm, GeGLU, CrossEntropyLoss         |
 | Qwen2       | `liger_kernel.transformers.apply_liger_kernel_to_qwen2`    | RoPE, RMSNorm, SwiGLU, CrossEntropyLoss, FusedLinearCrossEntropy        |
+| Phi3        | `liger_kernel.transformers.apply_liger_kernel_to_phi3`    | RoPE, RMSNorm, SwiGLU, CrossEntropyLoss         |

```

Comment:
Sorry for the whitespace modifications here, this is the only change in this file

Reply:
thanks a lot for helping on the housekeeping! <3

Reply:
do you want to implement for FusedLinearCrossEntropy in the next PR and also create a issue to track? Essentially add a method patch https://github.com/linkedin/Liger-Kernel/blob/8aab06a59cffa0d614bd2e10a9726ec680b3f294/src/liger_kernel/transformers/model/llama.py#L25

Reply:
Yeah happy to do that as well

Reply:
https://github.com/linkedin/Liger-Kernel/issues/98